### PR TITLE
chore: add install & usage info to README

### PR DESCRIPTION
**Pull request by yusefnapora**
Still a few things left to do, but I've got to go AFK for an hour or so, so might as well push this first :) Should be complete by EoD, barring mishap.

### CODE REVIEW

**Comment by yusefnapora** [comment]
@ice-breaker-tg question about the `upload` method... can I give it any `bytes`, or does it need to be CAR data?

**Comment by yusefnapora** [comment]
Okay, I think this covers the current state of things pretty well. @ice-breaker-tg let me know if I'm telling any lies 😄

**Comment by ice-breaker-tg** [comment]
> @ice-breaker-tg question about the `upload` method... can I give it any `bytes`, or does it need to be CAR data?

It must be car data for now, but yeah we can sort out how to do the car creation stuff.

**Review by ice-breaker-tg — APPROVED** [review]
This all looks great!

Thanks for taking the time to clarify / clean up docs here.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,173 @@
...
+
+And here's an example of calling `createClient` with the correct values for the production services:
+
+```js
+import { createClient } from 'w3up-client'
+
+const client = createClient({
+  serviceDID: 'did:key:z6MkrZ1r5XBFZjBU34qyD8fueMbMRkKw17BZaq2ivKFjnz2z',
+  serviceURL 'https://8609r1772a.execute-api.us-east-1.amazonaws.com',
+  accessDID: 'https://access-api.web3.storage',
```
Comment (ice-breaker-tg) [inline README.md:68]: These two should be switched (url and DID) for access.

**Pull request merged**